service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/ChildSubmissionController.java

```diff
@@ -41,17 +56,44 @@ public AboutToStartOrSubmitCallbackResponse handleMidEvent(@RequestBody Callback
             .build();
     }
 
+    @PostMapping("/about-to-submit")
+    public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody CallbackRequest callbackRequest) {
+        CaseDetails caseDetails = callbackRequest.getCaseDetails();
+
+        return childrenMigrationService.addHiddenValues(caseDetails);
+    }
+
     @SuppressWarnings("unchecked")
     private List<String> validate(CaseDetails caseDetails) {
         ImmutableList.Builder<String> errors = ImmutableList.builder();
 
-        Map<String, Object> childrenData = (Map<String, Object>) caseDetails.getData().get("children");
-        Children children = mapperService.mapObject(childrenData, Children.class);
-        if (children.getAllChildren().stream()
-            .map(Child::getChildDOB)
-            .filter(Objects::nonNull)
-            .anyMatch(dateOfBirth -> dateOfBirth.after(new Date()))) {
-            errors.add("Date of birth cannot be in the future");
+        if (caseDetails.getData().containsKey("children1")) {
```

Comment:
Do not operate on maps, deserialise to an object instance and operate on it.

Sam got away without doing it as we want to put in under test as soon as possible but rest PRs needs to be in good order before merge. Ps. Sam still needs to refactor his code, just in separate PR.